terraform/modules/gro_ingestion_service/lambdas/test/const/dbItem.ts

```diff
@@ -0,0 +1,53 @@
+export const dbItem = {
+    hash: {
+        S: "abc",
+    },
+    RegistrationId: {
+        S: "111",
+    },
+    EventTime: {
+        S: "2023-01-11",
+    },
+    DateOfDeath: {
+        S: "2023-01-01",
+    },
+    FirstForename: {
+        S: "Forename",
+    },
+    Surname: {
+        S: "Surname",
+    },
+    Sex: {
+        S: "Male",
+    },
+    MaidenSurname: {
+        S: null,
```

Comment:
I think these should be empty string not null, as we will want to make sure the record is returned with that and is stored with that